packages/providers/src/mina-node/utils/json-serializer.ts

```diff
@@ -0,0 +1,3 @@
+import JSONbig from 'json-bigint'
```

Comment:
For JSON serialization with extra types we use `superjson`. It appears in other packages.

Reply:
This is just a copy of the old `mina-explorer` sub-package, I will work on switching to `superjson` in another PR. Starting small.